src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsTests/TestSuiteStepsFlow.tsx

```diff
@@ -0,0 +1,106 @@
+import {useEffect, useMemo, useState} from 'react';
+import ReactFlow, {Controls, Edge, Node} from 'reactflow';
+import 'reactflow/dist/style.css';
+
+import {nanoid} from '@reduxjs/toolkit';
+
+import {LocalStep} from '@src/models/testSuite';
+
+import IntersectionNode from './Nodes/IntersectionNode';
+import StepNode from './Nodes/StepNode';
+import {ReactFlowContainer} from './SettingsTests.styled';
+
+interface ExtendedNode extends Node {
+  group?: number | string;
+}
+
+type TestSuiteStepsFlowProps = {
+  steps: LocalStep[][];
```

Comment:
Isn't `stopOnFailure` on group level, not each of tests? Like:
```
{
  execute: [ { test: 'some-name' }, { test: 'other-name' } ],
  stopOnFailure: true
}
```

Reply:
It is on step level, yes
<img width="308" alt="image" src="https://github.com/kubeshop/testkube-dashboard/assets/25755402/302de8a5-3a47-4504-b843-dd051eb5457b">

Reply:
In that case this `TestSuiteStepsFlowProps['steps']` should be redesigned, right? As it expects data in format:

```ts
[
  [
    {stopTestOnFailure: true, test: 'test-name'},
    {stopTestOnFailure: true, test: 'test-name-2'},
    {delay: '50s'},
  ],
  [
    {stopTestOnFailure: true, test: 'test-name'},
    {stopTestOnFailure: true, test: 'test-name-2'},
    {delay: '50s'},
  ],
]
```

Reply:
No, `LocalStep` doesn't have anything related to `stopOnFailure`. It is on another level

Reply:
Right, sorry! I don't know how I checked it twice and seen `stopOnFailure` in `TestSuiteStep`, not in `TestSuiteBatchStep` 🙂